File: Assets/VirtualCity/ProtoDefine/Scence.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public  class Scence {
[ProtoMember(1)]
    public Dictionary<long?, PlayerStatus> playerStatuses;
[ProtoMember(2)]
    public string scenceId;

    public Dictionary<long?, PlayerStatus> getPlayerStatuses() {
        return playerStatuses;
    }

    public void setPlayerStatuses(Dictionary<long?, PlayerStatus> playerStatuses) {
        this.playerStatuses = playerStatuses;
    }

    public string getScenceId() {
        return scenceId;
    }

    public void setScenceId(string scenceId) {
        this.scenceId = scenceId;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/building/Scence.java
package com.kingston.jforgame.server.game.building.entity;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

import java.util.Map;

public  class Scence {
    @Protobuf(order = 91,fieldType = FieldType.MAP)
    private Map<Long, PlayerStatus> playerStatuses;
    @Protobuf(order = 92)
    private String scenceId;

    public Map<Long, PlayerStatus> getPlayerStatuses() {
        return playerStatuses;
    }

    public void setPlayerStatuses(Map<Long, PlayerStatus> playerStatuses) {
        this.playerStatuses = playerStatuses;
    }

    public String getScenceId() {
        return scenceId;
    }

    public void setScenceId(String scenceId) {
        this.scenceId = scenceId;
    }
}
